libs/cuvslam/cuvslam2.cpp

```
@@ -176,9 +183,15 @@ void SetTrackerRigAndIntrinsics(std::vector<std::unique_ptr<camera::ICameraModel
 void CheckCameras(const cuvslam::Rig& rig) {
   THROW_INVALID_ARG_IF(rig.cameras.empty(), "No cameras in a rig");
   THROW_INVALID_ARG_IF(rig.cameras.size() > camera::Rig::kMaxCameras, "Number of cameras limit exceeded");
+  const auto is_finite = [](float v) { return std::isfinite(v); };
```

Clang (14.0.6)

[warning] parameter name 'v' is too short, expected at least 3 characters

(readability-identifier-length)